docs/book/common_code/node_controller.md

```diff
@@ -1,5 +1,8 @@
 # Node Controller
 
+> NOTE: This controller is going to be deprecated in v1alpha2,
```

Comment:
Would it be ok to remove the node controller now, instead of saying it's deprecated?

Reply:
Probably not, I know for example that the vSphere provider was relying on it at one point.

Reply:
Agreed, we should remove the node controller in v1alpha2. It's actually already part of #1051

Reply:
It might be better to actually call this out as deprecated now, and state that it will be removed in v1alpha2, otherwise some may interpret this as it will still exist in a deprecated state for v1alpha2.

Reply:
Good point, just reworded

Reply:
Does it need to remain in `master` now, though?

Reply:
@davidewatson will probably have a better answer, but I think that the gitbook is served from master

Reply:
@vincepri is correct, the book is served from master. To see a preview you can click the details of the `deploy/netlify` stage...